src/battle_anim.c

```diff
@@ -509,6 +634,68 @@ static void Cmd_createvisualtask(void)
     gAnimVisualTaskCount++;
 }
 
+extern u32 GetBattlerMoveTargetType(u8 battlerId, u16 move);
```

Comment:
I think it would be better to just include the needed header at the top instead of extern.